### ISSUE TITLE
[Feature] Provide a way to use a normal window title bar

### ISSUE THREAD

**Issue by odalet** [sourcegit-scm/sourcegit#390]
Hi,

The application is great and it looks nice, especially the custom chrome with the tabs in the title bar; however this does not play nice with WSL (I don't know if it poses issues with real Linux environments though):

The Desktop Manager that comes with WSL is very, very barebones and when I maximize the window, the application becomes unusable: mouse coordinates are all shifted as if a normal title bar were there even though it's not. For example, if I want to cloick the settings icon, I have to place my louse cursor like that:

![image](https://github.com/user-attachments/assets/2a702b7a-09ec-418c-8037-d1e62a8914ae)

With this weird shift, I can't close, restore or minimize the application as the mouse should be off screen in order to do so...
And once I've maximized the window, I have to edit the json configuration file by hand and tweak `LauncherWindowState`...

Long story short, could you provide a setting that uses a regular DM-provided title bar (VS Code provides this for Linux by the way), so that I can use it from WSL?

PS: there are many known issues / work in progress in the Avalonia source code aimed at fully supporting custom window chromes on Linux (I think that at the moment, this is only supported for Windows, and maybe macos...)

**Comment by love-linger** [comment]
This is an issue about `WSLg` that affects all the borderless windows, and it does not exist in real Linux environment (and other platforms).

For example, VSCode with `custom` title bar enabled in WSL:
![image](https://github.com/user-attachments/assets/516d6051-2713-4084-a48b-da886a9e7dee)

I'll try to add an option in preference to use native window frame.

**Comment by love-linger** [comment]
![image](https://github.com/user-attachments/assets/226eae7f-2490-45d6-990f-0abdcab3f98c)

You can try it from https://github.com/sourcegit-scm/sourcegit/actions/runs/10501913361

**Comment by odalet** [comment]
That was both quick and exactly what I need. Thank you so much for this!

**Comment by faienz93** [comment]
If anyone still has this problem, and as in my case, cannot open preferences, you can edit `~/.config/SourceGit/preference.json` by putting the value:

```
"UseSystemWindowFrame": true
```
